static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import fetch from 'lib/fetch';
+
+const isCheckbox = element => element.type === 'checkbox';
+const isValid = element => element.name !== '';
```

Comment:
On a separate note, I think the name `isValid` along with this implementation deserves a comment to describe its purpose. It's not clear why any element with a name is "valid". Could it be called _isNamed_ for example?

Reply:
Good idea